src/tree/assess/assessments/Assessment.js

```diff
@@ -9,16 +9,42 @@ class Assessment {
 	/**
 	 * Creates a new assessment.
 	 *
-	 * @param {string} [name] An optional name.
+	 * @param {string}                              name       The name to give this assessment.
+	 * @param {module:tree/research.TreeResearcher} researcher The researcher to do researches with.
 	 *
 	 * @abstract
 	 */
-	constructor( name ) {
+	constructor( name, researcher ) {
 		/**
-		 * This assessment's unique name.
+		 * This assessment's name.
 		 * @type {string}
 		 */
 		this.name = name;
+		/**
+		 * The researcher to do researches with.
+		 * @type {module:tree/research.TreeResearcher}
+		 */
+		this.researcher = researcher;
```

Comment:
Since this has a getter and a setter, I think it makes more sense to prefix this with an underscore.

Reply:
Done! 🙂